aix/SPECS/wazuh-agent-aix.spec

```diff
@@ -290,7 +290,7 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/wodles/*
 
 %changelog
-* Thu Sep 28 2023 support <info@wazuh.com> - 4.5.3
+* Thu Oct 03 2023 support <info@wazuh.com> - 4.5.3
```

Comment:
It's Tuesday not Thursday

Reply:
Done 
https://github.com/wazuh/wazuh-packages/pull/2484/commits/def1f28187037bf496ca9b4380a0bf082908e00a